### PR TITLE
VIH-6539 Mute and Umute issues

### PR DESCRIPTION
**Pull request by jdssunil**
### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/VIH-6539

### Change description ###
Fix for mute

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ x] No
```

### CODE REVIEW

**Review by shaed-parkar — COMMENTED** [review]
Could you make sure the tests associated to this are updated to assert on this scenario please? thanks

File: VideoWeb/VideoWeb/ClientApp/src/app/waiting-space/participants-panel/participants-panel.component.spec.ts [hunk 1]
```diff
@@ -201,6 +201,40 @@ describe('ParticipantsPanelComponent', () => {
...
+        // Unmute the participant
+        component.toggleMuteParticipant(pat);
+
+        expect(videocallService.muteAllParticipants).toHaveBeenCalledTimes(0);
+    });
+
+    it('should not mute conference when any of the second last participant is muted manually', () => {
+        videocallService.muteAllParticipants.calls.reset();
+        component.isMuteAll = true;
+        // Mute all participants except one
```
Comment (shaed-parkar) [inline VideoWeb/VideoWeb/ClientApp/src/app/waiting-space/participants-panel/participants-panel.component.spec.ts:224]: this comment does not reflect the setup here.

File: VideoWeb/VideoWeb/ClientApp/src/app/waiting-space/participants-panel/participants-panel.component.spec.ts [hunk 2]
```diff
@@ -201,6 +201,40 @@ describe('ParticipantsPanelComponent', () => {
...
+        for (let index = 0; index < component.participants.length - 1; index++) {
+            component.participants[index].isMuted = false;
+            (<ParticipantPanelModel>component.participants[index]).status = ParticipantStatus.InHearing;
+        }
+
+        // Get any muted participant
+        const pat = component.participants[0];
```
Comment (shaed-parkar) [inline VideoWeb/VideoWeb/ClientApp/src/app/waiting-space/participants-panel/participants-panel.component.spec.ts:231]: You should filter for participants with the muted boolean set to true to make this test robust

File: VideoWeb/VideoWeb/ClientApp/src/app/waiting-space/participants-panel/participants-panel.component.spec.ts [hunk 3]
```diff
@@ -201,6 +201,40 @@ describe('ParticipantsPanelComponent', () => {
...
+        videocallService.muteAllParticipants.calls.reset();
+        component.isMuteAll = true;
+        // Mute all participants except one
+        for (let index = 0; index < component.participants.length - 1; index++) {
+            component.participants[index].isMuted = true;
+            (<ParticipantPanelModel>component.participants[index]).status = ParticipantStatus.InHearing;
+        }
+
+        // Get any muted participant
+        const pat = component.participants[0];
```
Comment (shaed-parkar) [inline VideoWeb/VideoWeb/ClientApp/src/app/waiting-space/participants-panel/participants-panel.component.spec.ts:214]: You should filter for participants with the muted boolean set to true to make this test robust

**Review by shaed-parkar — COMMENTED** [review]
The comments are a tad misleading, could you please update them to reflect the scenario. Thanks.

Otherwise nice work!

File: VideoWeb/VideoWeb/ClientApp/src/app/waiting-space/participants-panel/participants-panel.component.spec.ts (hunk 3 above)
Comment (jdssunil) [reply VideoWeb/VideoWeb/ClientApp/src/app/waiting-space/participants-panel/participants-panel.component.spec.ts]: You made the same mistake as mine....Copy and paste comments ;)

File: VideoWeb/VideoWeb/ClientApp/src/app/waiting-space/participants-panel/participants-panel.component.spec.ts (hunk 2 above)
Comment (jdssunil) [reply VideoWeb/VideoWeb/ClientApp/src/app/waiting-space/participants-panel/participants-panel.component.spec.ts]: Done

File: VideoWeb/VideoWeb/ClientApp/src/app/waiting-space/participants-panel/participants-panel.component.spec.ts (hunk 1 above)
Comment (jdssunil) [reply VideoWeb/VideoWeb/ClientApp/src/app/waiting-space/participants-panel/participants-panel.component.spec.ts]: Updated.

**Review by RuwanikaWeerakkody — APPROVED** [review]

**Pull request merged**